src/game_state/Platform.h

```diff
@@ -0,0 +1,23 @@
+// bnbeck
+
+#ifndef PLATFORM_H_
+#define PLATFORM_H_
+
+#include <glm/ext.hpp>
+
+#include "GameObject.h"
+
+#define PLATFORM "../Assets/shaders/platform.json"
+#define PLATFORM_MESH "../Assets/models/platform.obj"
+
+class Platform : public GameObject {
+ public:
+   Platform(glm::vec3 position);
+   Platform(glm::vec3 position, glm::vec3 scale);
```

Comment:
Again, why so many constructors? It makes it unclear to me as to what is necessary to create a platform